### PR TITLE
Enum branch

### PR DESCRIPTION
**Pull request by ealwafai**
- Changes Implemented:
  - Updated user poro to include a goals array
  - Updated new form with the goals array

- Quality Control Checklist:

  - [] Code adheres to Rubocop styling (if unavoidable infractions, please clarify below)
  - [] 100% SimpleCov test coverage (if below, please clarify below)
  - [] Last commit has passing Travis build

- Blockers (if applicable):
None
- Next Steps & Additional Notes:

### CODE REVIEW

File: app/services/back_end_service.rb [hunk 1]
```diff
@@ -31,8 +31,8 @@ def create_user(user_params)
     def base_url
       # NOTE: base_url needs to be localhost if you want to auth in during development
       # Open your BE server via $ rails s --port 4500
-      # 'http://localhost:4500'
-      'https://spotme-app-api.herokuapp.com'
+      'http://localhost:4500'
+      # 'https://spotme-app-api.herokuapp.com'
```
Comment (Scott-Borecki) [inline app/services/back_end_service.rb:35]: Let's update the `base_url` to be the backend API url before we merge this.

File: spec/services/back_end_service_spec.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe BackEndService do
   it 'can retrieve the base url for API calls' do
-    expect(BackEndService.base_url).to eq 'https://spotme-app-api.herokuapp.com'
+    expect(BackEndService.base_url.class).to eq String
   end
```
Comment (Scott-Borecki) [inline spec/services/back_end_service_spec.rb:6]: Creative workaround here!

**Review by Scott-Borecki — CHANGES_REQUESTED** [review]
Looks good to me!  Thanks for putting this together!  

Let's update the `base_url` to be the backend API url before we merge this.  After you update that, should be good to go.

File: app/services/back_end_service.rb (hunk 1 above)
Comment (tvaroglu) [reply app/services/back_end_service.rb]: I just updated this and rerecorded cassettes.  Please take approve so we can merge

**Review by Scott-Borecki — APPROVED** [review]

**Pull request merged**